core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -572,7 +572,7 @@ private[chisel3] object Builder extends LazyLogging {
     dynamicContext.whenStack = s
   }
 
-  def currentWhen(): Option[WhenContext] = dynamicContext.whenStack.headOption
+  def currentWhen: Option[WhenContext] = dynamicContext.whenStack.headOption
```

Comment:
why does this one not need a deprecation version?

Reply:
Same package private thing, doesn't need to be deprecated.